### PR TITLE
Contributors cannot be listed with wildcard tag searches(Resolves#1921)

### PR DESCRIPTION
**Pull request by SidharthBansal**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/views/tag/show.html.erb [hunk 1]
```diff
@@ -57,7 +57,11 @@
...
     <li<% if @node_type == "wiki" %> class="active"<% end %>><a href="/wiki/tag/<%= params[:id] %>"><i class="fa fa-book"></i> <%= raw t('tag.show.wiki_pages') %></a></li>
     <li<% if @node_type == "maps" %> class="active"<% end %>><a href="/maps/tag/<%= params[:id] %>"><i class="fa fa-map-marker"></i> <%= t('tag.show.maps') %></a></li>
-     <li<% if @node_type == "contributors" %> class="active"<% end %>><a href="/contributors/<%= params[:id] %>"><i class="fa fa-user"></i> <%= raw t('tag.show.contributors') %> <span class="badge"><%=@length %></span></a></li>
+    <% if !@wildcard %>
+        <li<% if @node_type == "contributors" %> class="active"<% end %>><a href="/contributors/<%= params[:id] %>"><i class="fa fa-user"></i> <%= raw t('tag.show.contributors') %> <span class="badge"><%=@length %></span></a></li>
+    <% else %>
+        <li<% if @node_type == "contributors" %> class="active"<% end %>><a href="#" rel="tooltip" title="Contributors cannot be listed for wildcard tag searches" ><i class="fa fa-user"></i> <%= raw t('tag.show.contributors') %></a></li>
```
Comment (jywarren) [inline app/views/tag/show.html.erb]: This looks good; shall we also add a `class="disabled"` to the `<li>` so it appears faded? Or could we simple delete the tab? Thanks for looking into this!!

Comment (SidharthBansal) [reply app/views/tag/show.html.erb]: Faded is better. As otherwise a user may think where has the contributors tab gone away all of a sudden?

**Comment by jywarren** [comment]
I agree. I think `disabled` will fade it out!

**Comment by SidharthBansal** [comment]
I have added.
It is working great!!!

**Comment by jywarren** [comment]
🎉 🎈 Great work!

**Pull request merged**